src/VisualStudio/Setup.Next/source.extension.vsixmanifest

```diff
@@ -11,6 +11,11 @@
     <InstallationTarget Version="[15.0,]" Id="Microsoft.VisualStudio.VWDExpress" />
     <InstallationTarget Version="[15.0,]" Id="Microsoft.VisualStudio.VSWinExpress" />
   </Installation>
+  <Installer>
+    <Actions>
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.EditorFeatures.Next.dll" />
+    </Actions>
+  </Installer>
   <Dependencies>
```

Comment:
This VSIX (Setup.Next) doesn't have <ShortcutPath> specified. That means this will be extracted into a folder with mangled name, correct? Will ngen work in this case (i.e. don't we need well known path for ngen)?